packages/flame/lib/src/effects/sequence_effect.dart

```diff
@@ -66,17 +66,12 @@ class SequenceEffect extends Effect {
             infinite: infinite,
             repeatCount: repeatCount,
           ),
-        );
-
-  final List<Effect> _effects;
-
-  @override
-  void onMount() {
-    super.onMount();
-
+        ) {
     addAll(_effects);
   }
 
+  final List<Effect> _effects;
```

Comment:
Is this list needed, they are stored within the controller right?

Reply:
nope! removed.